.github/workflows/continuous-integration.yml

```
@@ -45,8 +45,9 @@ jobs:
         echo "pytag=${{ env.REGISTRY }}/stac-utils/pgstac-pyrust:$ref" >>$GITHUB_OUTPUT;
         echo "buildpy=$buildpg" >>$GITHUB_OUTPUT;
 
+  # This builds a base postgres image that has everything installed to be able to run pgstac. This image does not have pgstac itself installed.
   buildpg:
-    name: Build and push base postgres
+    name: Build and push base postgres image (you will need to run pypgstac migrate to install pgstac)
```

```suggestion
    name: Build and push base postgres image
```

I don't think we need this, It's fine with the comment 👍